server/network.go

```diff
@@ -149,14 +152,26 @@ func newLocalNetwork(opts localNetworkOptions) (*localNetwork, error) {
 			return nil, fmt.Errorf("failed merging provided configs: %w", err)
 		}
 
-		cfg.NodeConfigs[i].ConfigFile, err = createConfigFileString(mergedConfig, logLevel, logDir, dbDir, opts.pluginDir, opts.whitelistedSubnets)
+		cfg.NodeConfigs[i].ConfigFile, err = createConfigFileString(mergedConfig, nodeLogLevel, logDir, dbDir, opts.pluginDir, opts.whitelistedSubnets)
+		if err != nil {
+			return nil, err
+		}
+
+		if cfg.NodeConfigs[i].CChainConfigFile == "" {
+			cfg.NodeConfigs[i].CChainConfigFile = "{}"
+		}
+		cchainLogLevel, err := logutil.AvalanchegoToCorethLogLevel(nodeLogLevel)
```

Comment:
Why are we changing this only item for the cchain config? It feels like a bit arbitrary just only doing this.

Maybe we should consider the whole cchain (and other custom chain configs?) more globally, as a whole?

Reply:
The PR is trying to manage node logs considering only avalanchego+coreth. Usually the common case for the moment. You need to pass cchain config to coreth to get its log level modified. Probably be need another PR for changing all the other cchain config using flags. For other plugins, this seems out of scope, refers to avalanchego pluging stdout/stderr log management/filtering.

Reply:
https://github.com/ava-labs/avalanche-network-runner/pull/149/files#r872665682